modules/ossm-enabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,275 @@
+----
+$ oc get pods -n bookinfo
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                             READY   STATUS    RESTARTS   AGE
+details-v1-65cfcf56f9-gm6v7      1/1     Running   0          4m55s
+productpage-v1-d5789fdfb-8x6bk   1/1     Running   0          4m53s
+ratings-v1-7c9bd4b87f-6v7hg      1/1     Running   0          4m55s
+reviews-v1-6584ddcf65-6wqtw      1/1     Running   0          4m54s
+reviews-v2-6f85cb9b7c-w9l8s      1/1     Running   0          4m54s
+reviews-v3-6f5b775685-mg5n6      1/1     Running   0          4m54s
+----
+
+. Run the following command at the CLI to apply the injection label to the `bookinfo` namespace:
++
+[source,terminal]
+----
+$ oc label namespace bookinfo istio-injection=enabled
+namespace/bookinfo labeled
+----
+
+. Redeploy existing workloads in the `bookinfo` namespace to enable sidecar injection. Use the following command to perform a rolling update of all workloads:
++
+[source,terminal]
+----
+$ oc -n bookinfo rollout restart deployment
+----
+
+.Verification
+
+. Verify the rollout by checking that the new pods display `2/2` containers as `READY`, confirming successful sidecar injection.
++
+[source,terminal]
+----
+$ oc get pods -n bookinfo
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                              READY   STATUS    RESTARTS   AGE
+details-v1-7745f84ff-bpf8f        2/2     Running   0          55s
+productpage-v1-54f48db985-gd5q9   2/2     Running   0          55s
+ratings-v1-5d645c985f-xsw7p       2/2     Running   0          55s
+reviews-v1-bd5f54b8c-zns4v        2/2     Running   0          55s
+reviews-v2-5d7b9dbf97-wbpjr       2/2     Running   0          55s
+reviews-v3-5fccc48c8c-bjktn       2/2     Running   0          55sz
+----
+
+[id="ossm-enabling-sidecar-injection-exclude-workload-from-mesh_{context}"]
+== Exclude a workload from the mesh
+
+You can exclude specific workloads from sidecar injection within a namespace where injection is enabled for all workloads.
+
+[NOTE]
+====
+This example is for demonstration purposes only. The bookinfo application requires all workloads to be part of the mesh for proper functionality.
+====
+
+.Procedure
+
+. Open the application's `Deployment` resource in an editor. In this case, exclude the `ratings-v1` service.
+
+. Modify the `spec.template.metadata.labels` section of your `Deployment` resource to include the label `sidecar.istio.io/inject: false` to disable sidecar injection.
++
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+name: ratings-v1
+namespace: bookinfo
+labels:
+  app: ratings
+  version: v1
+spec:
+  template:
+    metadata:
+      labels:
+        sidecar.istio.io/inject: 'false'
+----
++
+[NOTE]
+====
+Adding the label to the `Deployment`'s top-level `labels` section will not impact sidecar injection.
```

Comment:
It looks like something went wrong here with the quotes in the preview. It looks like the second ` was not picked up possibly because of the apostrophe, I am not sure.

Reply:
I have changed the sentence and made it without apostrophe to avoid confusion.